libs/sdk-ui-tests/stories/_infra/storyGroups.ts

```diff
@@ -7,3 +7,4 @@ export const EmbeddingStories = "05 Stories For Embedding using InsightView";
 export const FilterStories = "10 Filters";
 export const ConfigurationControls = "11 Configuration Controls";
 export const UiKit = "12 UI Kit";
+export const WebComponents = "13 Web Components";
```

Comment:
Web components section is rendered at the very end of the storybook. It's a bit inconsistent with the rest of the Storybook sections, that are ordered reversed (12 ... 1). Will try to find a way to order it correctly.